addons/safemode/functions/fn_setWeaponSafety.sqf

```diff
@@ -0,0 +1,33 @@
+/*
+ * Author: Brostrom.A
+ * Safe or unsafe the given weapon based on weapon state; locked or unlocked.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: Weapon <STRING>
+ * 2: State <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [ACE_player, true] call ace_safemode_fnc_setWeaponSafety
+ *
+ * Public: no
+ */
+
+params [
+    ["_unit", objNull, [objNull]], 
```

Comment:
```suggestion
    ["_unit", objNull, [objNull]],
```
White space at end of line :)

Reply:
There still is a space at the end of the line.

Reply:
now it should be merged.